api/assets/resources.json

```diff
@@ -546,6 +546,36 @@
         }
       }
     },
+    "SingleAgent": {
```

Comment:
I added the `SingleAgent` schema due to the slight differences between the agent data in /properties and /agents